### DISCUSSION TITLE
How to Create Steam Player Summaries With Next.js - Jagad Yudha Awali

### DISCUSSION THREAD

**Discussion by giscus** [jagadyudha/jagad.dev#26] [Announcements]
# How to Create Steam Player Summaries With Next.js - Jagad Yudha Awali

Using Steamworks API to display information about gaming activities on websites

http://localhost:3000/posts/how-to-create-steam-player-summaries-with-next-js

**Comment by jagadyudha** [comment]
Please leave comments and feedback on this post 😀
